From 46759fbe61fcca60bc4e619a9cde0977f48c0723 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Tue, 22 Jan 2013 14:28:32 +0000
Subject: [PATCH 140/364] * util/grub-reboot.in (usage): Document the need for
 GRUB_DEFAULT=saved. * util/grub-set-default.in (usage): Likewise. Reported
 by: Brian Candler.  Fixes Ubuntu bug #1102925.

---
 ChangeLog                | 7 +++++++
 util/grub-reboot.in      | 2 ++
 util/grub-set-default.in | 2 ++
 3 files changed, 11 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index f3a9fa0..8c4d087 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,10 @@
+2013-01-22  Colin Watson  <cjwatson@ubuntu.com>
+
+	* util/grub-reboot.in (usage): Document the need for
+	GRUB_DEFAULT=saved.
+	* util/grub-set-default.in (usage): Likewise.
+	Reported by: Brian Candler.  Fixes Ubuntu bug #1102925.
+
 2013-01-21  Vladimir Serbinenko  <phcoder@gmail.com>
 
 	* grub-core/lib/libgcrypt_wrap/cipher_wrap.h: Include sys/types.h rather
diff --git a/util/grub-reboot.in b/util/grub-reboot.in
index 7516a03..1a91d36 100644
--- a/util/grub-reboot.in
+++ b/util/grub-reboot.in
@@ -20,6 +20,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 bindir=@bindir@
+sysconfdir="@sysconfdir@"
 PACKAGE_NAME=@PACKAGE_NAME@
 PACKAGE_VERSION=@PACKAGE_VERSION@
 datarootdir="@datarootdir@"
@@ -45,6 +46,7 @@ export TEXTDOMAINDIR="@localedir@"
 usage () {
     gettext_printf "Usage: %s [OPTION] MENU_ENTRY\n" "$self"
     gettext "Set the default boot menu entry for GRUB, for the next boot only."; echo
+    gettext_printf "This requires setting GRUB_DEFAULT=saved in %s/default/grub.\n" "$sysconfdir"
     echo
     print_option_help "-h, --help" "$(gettext "print this message and exit")"
     print_option_help "-v, --version" "$(gettext "print the version information and exit")"
diff --git a/util/grub-set-default.in b/util/grub-set-default.in
index 443e56f..ea18da1 100644
--- a/util/grub-set-default.in
+++ b/util/grub-set-default.in
@@ -20,6 +20,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 bindir=@bindir@
+sysconfdir="@sysconfdir@"
 PACKAGE_NAME=@PACKAGE_NAME@
 PACKAGE_VERSION=@PACKAGE_VERSION@
 datarootdir="@datarootdir@"
@@ -45,6 +46,7 @@ export TEXTDOMAINDIR="@localedir@"
 usage () {
     gettext_printf "Usage: %s [OPTION] MENU_ENTRY\n" "$self"
     gettext "Set the default boot menu entry for GRUB."; echo
+    gettext_printf "This requires setting GRUB_DEFAULT=saved in %s/default/grub.\n" "$sysconfdir"
     echo
     print_option_help "-h, --help" "$(gettext "print this message and exit")"
     print_option_help "-v, --version" "$(gettext "print the version information and exit")"
-- 
1.8.1.4

